### ISSUE TITLE
Is this exercise in IBP meant to be here?

### ISSUE THREAD

**Issue by Alex-Jordan** [APEXCalculus/APEXCalculusPTX#160]
This exercise:
Exercise 9 
https://opentext.uleth.ca/apex-video/sec_IBP.html#exercise-1010

Is something where it's natural to use substitution. At the risk of embarrassing myself, I don't see how to do it with integration by parts. So is it meant to be here in the IBP exercises? A perfectly good answer is yes, to keep students on their toes and not have IBP blinders on. But it's something I want to cull from my assignment for this section. If I wasn't scrutinizing the exercises, I might have not noticed.

**Comment by Alex-Jordan** [comment]
I guess there are a few scattered in here that may be meant to keep everyone on their toes. Like 32.

And 30 and 31 _could_ be done with IBP, but substitution seems wiser.

**Comment by APEXCalculus** [comment]
Your guess is right, #9 is intentional. In this case, the point was less
"keep them on their toes" than demonstrate how two similar-looking
integrals need vastly different integration techniques to evaluate them.

#9 looks like many other IBP setups, including examples previously given: x
* f(x), where f(x) is a trig/exponential/other function.

And #32 is also intentional; I expect some students will labor over this
for a few minutes then do a dope-slap as they recognize the obvious. It
then leads into #33 & 34.

#30 & 31 are less about keeping on your toes than recognizing alternate
ways of handling things. IBP in #30 still requires substitution, and once
that is recognized, we may see that skipping IBP altogether is best & just
use substitution, which is then what you really need in #31.

Hope that helps.

**Comment by sean-fitzpatrick** [comment]
For some reason more Gmail has been putting more of these in my spam.
Fortunately I checked.

Maybe I'm not supposed to reply: I'm currently on strike.

I had a few students in my engineering calculus class who insisted on using
integration by parts for the integral of sin(nx)sin(mx).

Surprisingly, they succeeded. But it was a lot of work and you get a really
ugly result.

This made me suspicious that maybe they had colluded on their test (we're
currently online) but since they'd done a page of work to replace a one
line trig identity I decided they'd already suffered enough.
